ui/src/main.rs

```diff
@@ -366,6 +366,13 @@ struct EvaluateResponse {
     error: Option<String>,
 }
 
+#[derive(Debug, Clone, Deserialize)]
+struct NoWebSocketRequest {}
+
+#[derive(Debug, Clone, Serialize)]
+struct NoWebSocketResponse {}
+
+
```

Comment:
```suggestion
#[derive(Debug, Clone, Deserialize)]
struct NoWebSocketRequest;

#[derive(Debug, Clone, Serialize)]
struct NoWebSocketResponse;

```

(note the removal of the extra newline)